### DISCUSSION TITLE
why my project can't work correctly

### DISCUSSION THREAD

**Discussion by manyuemeiquqi** [vuejs/vitepress#2145] [Q&A] [answered]
why my project can't work correctly

my github link https://github.com/jiangyeka/jiangyeka.github.io

https://github.com/jiangyeka/jiangyeka.github.io/actions/runs/4531960072/jobs/7982742838#step:17:28

when i push ,git flow work failed 

i copy .yml from vitepress site

can you help me find the reason 
![image](https://user-images.githubusercontent.com/54520846/227931646-6f86c0fe-647c-4598-be56-f171fd31707e.png)

**Comment by brc-dd** [comment, marked as answer]
Change this line:

https://github.com/jiangyeka/jiangyeka.github.io/blob/main/.github/workflows/static.yml#L30

You don't have `docs/` directory, you've everything at root, so you simply need: `path: '.vitepress/dist'`
